voltdbclient/url.go

```diff
@@ -0,0 +1,33 @@
+package voltdbclient
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+)
+
+var errBadURL = errors.New("voltdb: bad connection url")
+
+func parseURL(src string) (*url.URL, error) {
+	u, err := url.Parse(src)
+	if err != nil {
+		return nil, err
+	}
+	if u.User == nil {
+		u.User = &url.Userinfo{}
+	}
+	if u.Scheme != "voltdb" {
```

Comment:
Should we consider it an error if the `u.Scheme` is not "voltdb" and it's not empty?